package.json

```diff
@@ -42,5 +46,16 @@
     "dependencies": {
         "react": "^16.13.1"
     },
-    "types": "dist/index.d.ts"
+    "types": "dist/index.d.ts",
+    "config": {
+        "commitizen": {
+            "path": "./node_modules/cz-conventional-changelog"
+        }
+    },
+    "husky": {
+        "hooks": {
+            "pre-commit": "yarn test-once && yarn lint",
```

Comment:
Maybe only `yarn lint` for pre-commit? Might get annoying when you want to make a WIP commit on a feature branch, I'd say pre-push is enough.